app/views/lists/_new_list.erb

```diff
@@ -0,0 +1,29 @@
+  <h2>Create a new list:</h2>
+
+  <% if @errors %>
```

Comment:
It would be cleaner to make an errors partial, since you use it in multiple view pages